### PR TITLE
Update ocp4-workload-quarkus-workshop to upgrade OCP 4.6.4

### PR DESCRIPTION
**Pull request by danieloh30**
##### SUMMARY
Update ocp4-workload-quarkus-workshop to upgrade OCP 4.6.4

##### ISSUE TYPE

- Feature Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: ansible/roles/ocp4-workload-quarkus-workshop/tasks/workload.yml [hunk 1]
```diff
@@ -15,6 +15,17 @@
...
+- name: Create CatalogSource Index 4.6
+  k8s:
+    state: present
+    merge_type:
+    - strategic-merge
+    - merge
+    definition: "{{ lookup('file', item ) | from_yaml }}"
+  loop:
+  - ./files/redhat-operators-index.yaml
+  - ./files/community-operators-index.yaml
```
Comment (jkupferer) [inline ansible/roles/ocp4-workload-quarkus-workshop/tasks/workload.yml:27]: No need to have `./files/` with `lookup('file', ... )`. It by default looks there.

File: ansible/roles/ocp4-workload-quarkus-workshop/files/stack.Dockerfile [hunk 2]
```diff
@@ -1,13 +1,13 @@
 # To build this stack:
 # docker build -t quay.io/username/quarkus-workshop-stack:VVV -f stack.Dockerfile .
-# docker push quay.io/username/quarkus-workshop-stack:VVVV
+# docker push quay.io/username/quarkus-workshop-stack:VVVV
```
Comment (jkupferer) [inline ansible/roles/ocp4-workload-quarkus-workshop/files/stack.Dockerfile:3]: Added dangling whitespace?

**Review by jkupferer — CHANGES_REQUESTED** [review]

File: ansible/roles/ocp4-workload-quarkus-workshop/files/stack.Dockerfile (hunk 2 above)
Comment (danieloh30) [reply ansible/roles/ocp4-workload-quarkus-workshop/files/stack.Dockerfile:3]: It's for creating a new PR without any changes.

File: ansible/roles/ocp4-workload-quarkus-workshop/tasks/workload.yml (hunk 1 above)
Comment (danieloh30) [reply ansible/roles/ocp4-workload-quarkus-workshop/tasks/workload.yml:27]: Should it Ok with?
  loop:
  - redhat-operators-index.yaml
  - community-operators-index.yaml

Comment (danieloh30) [reply ansible/roles/ocp4-workload-quarkus-workshop/tasks/workload.yml]: Updated

File: ansible/roles/ocp4-workload-quarkus-workshop/files/stack.Dockerfile (hunk 2 above)
Comment (danieloh30) [reply ansible/roles/ocp4-workload-quarkus-workshop/files/stack.Dockerfile]: removed the white sapce.

**Review by jkupferer — APPROVED** [review]

**Pull request merged**